stable/graylog/README.md

```diff
@@ -124,6 +124,7 @@ The following table lists the configurable parameters of the Cassandra chart and
 | `graylog.ingress.annotations`           | Graylog Ingress annotations                                                                                                                           | `{}`                                  |
 | `graylog.ingress.hosts`                 | Graylog Ingress host names                                                                                                                            | `[]`                                  |
 | `graylog.ingress.tls`                   | Graylog Ingress TLS configuration (YAML)                                                                                                              | `[]`                                  |
+| `graylog.ingress.extraPaths`            | Ingress extra paths to prepend to every host configuration. Useful when configuring [custom actions with AWS ALB Ingress Controller](https://kubernetes-sigs.github.io/aws-alb-ingress-controller/guide/ingress/*annotation/#actions). | `[]`                   |
```

Comment:
A link was broken. Should it be `https://github.com/kubernetes-sigs/aws-alb-ingress-controller/blob/master/docs/guide/ingress/annotation.md#actions`?

Reply:
I had an extra `*` in the url.
Going to fix. Thanks

Reply:
But, yes the content is the same. It's just that they have some pretty docs live.

Reply:
Fixed